stackit/internal/resources/argus/instance/actions.go

```diff
@@ -68,44 +63,22 @@ func (r Resource) Create(ctx context.Context, req resource.CreateRequest, resp *
 
 func (r Resource) createInstance(ctx context.Context, diags *diag.Diagnostics, plan *Instance) {
 	c := r.Provider.Client()
-	created := false
-	var got instances.Instance
-	if err := helper.RetryContext(ctx, common.DURATION_1H, func() *helper.RetryError {
-		if !created {
-			res, err := c.Argus.Instances.Create(ctx, plan.ProjectID.Value, plan.Name.Value, plan.PlanID.Value, map[string]string{})
-			if err != nil {
-				if common.IsNonRetryable(err) {
-					return helper.NonRetryableError(err)
-				}
-				return helper.RetryableError(err)
-			}
-			created = true
-			plan.ID = types.String{Value: res.InstanceID}
-		}
-
-		if plan.ID.Value == "" {
-			return helper.NonRetryableError(errors.New("received empty id"))
-		}
 
-		var err error
-		got, err = c.Argus.Instances.Get(ctx, plan.ProjectID.Value, plan.ID.Value)
-		if err != nil {
-			return helper.RetryableError(err)
-		}
+	_, process, err := c.Argus.Instances.Create(ctx, plan.ProjectID.Value, plan.Name.Value, plan.PlanID.Value, map[string]string{})
+	if err != nil {
+		diags.AddError("failed during instance creation", err.Error())
+		return
+	}
 
-		switch got.Status {
-		case consts.ARGUS_INSTANCE_STATUS_UPDATE_SUCCEEDED:
-			fallthrough
-		case consts.ARGUS_INSTANCE_STATUS_CREATE_SUCCEEDED:
-			return nil
-		case consts.ARGUS_INSTANCE_STATUS_CREATING:
-			return helper.RetryableError(fmt.Errorf("received status %s", got.Status))
-		}
+	res, err := process.Wait()
```

Comment:
Do we need to do `process.SetTimeout(1 * time.Hour)` to match the timeout of the old retry stuff? Same goes for all the other wait handlers, which sometimes have a timeout of 1 minute instead.